apps/chat-e2e/src/ui/webElements/attachFilesModal.ts

```diff
@@ -13,6 +13,10 @@ import { AttachFilesTree, Folders } from '@/src/ui/webElements/entityTree';
 import { FilesModalHeader } from '@/src/ui/webElements/filesModalHeader';
 import { Page } from '@playwright/test';
 
+export enum FileModalSection {
+  AllFiles = 'all files',
```

Comment:
let's use uppercase in names to match names on the UI

Reply:
done